app/src/scenes/phase1/components/TestimonialCard.js

```diff
@@ -19,7 +19,7 @@ const TestimonialCard = ({ title, description, picture, readingTime, isVideo = f
       </div>
       <div className="p-6">
         <h3 className="text-sm font-bold mb-3">{title}</h3>
-        <p className="text-sm">{description}</p>
+        <p className="text-[13px] text-[#3A3A3A]">{description}</p>
```

Comment:
On essaie de prendre le plus possible les presets tailwind (text-xs text-gray-700 par exp ici) plutôt que de redéfinir des paramètres, même si c'est pas pixel perfect sur la maquette. Je sais que c'était un retour PO ici mais on peut expliquer qu'on se tient à nos variables par souci d'homogénéité et d'efficacité.

Reply:
Je valide à 100%